.github/mergify.yml

```diff
@@ -0,0 +1,34 @@
+queue_rules:
+  - name: default
```

Comment:
Is the `name` value correct?

Reply:
It can be anything really, it's just an identifier. The mergify setup instructions call it that so I copied it. I can change it I think if you like.